edxval/management/commands/verify_pact.py

```diff
@@ -22,21 +22,30 @@ class Command(BaseCommand):
     """
     help = "Verify the VAL provider pacts"
 
+    default_opts = {
+            'broker_url': getattr(settings, 'PACT_BROKER_BASE_URL', None),
+            'publish_version': '1',
+            'publish_verification_results': getattr(settings, 'PUBLISH_VERIFICATION_RESULTS', False)
+        }
+
     def verify_pact(self):
         """
         Verify the pacts with Pact-verifier.
         """
-        # TODO: Use pact broker to get the pacts that specify VAL as the provider
         verifier = Verifier(
             provider='VAL',
             provider_base_url=settings.PROVIDER_BASE_URL
-
-        )
-        verifier.verify_pacts(
-            'edxval/pacts/vem-val.json',
-            provider_states_setup_url=settings.PROVIDER_STATES_URL,
         )
 
+        if self.default_opts['broker_url']:
+            verifier.verify_with_broker(
+                **self.default_opts,
+                verbose=False,
+                provider_states_setup_url=settings.PROVIDER_STATES_URL,
+            )
+        else:
+            log.error("Pact Broker Not Setup")
```

Comment:
Instead of logging error, use simple pact verification for now i.e. keep the existing code.